tests/json-fixtures/test_blockchain.py

```diff
@@ -116,7 +115,14 @@ def test_blockchain_fixtures(fixture_name, fixture):
     #     assert rlp.encode(genesis_header) == fixture['genesisRLP']
 
     db = MemoryDB()
-    evm = MainnetEVM.from_genesis(
+
+    evm = MainnetEVM
+    if fixture_name.startswith('Homestead'):
+        evm = EVM.configure(
+            'HomesteadEVM',
+            vm_configuration=[(0, HomesteadVM)])
```

Comment:
Sounds great.  Just leaving these things as I see them to try and keep both a record and a personal understanding of where some of the "bodies are buried".  Not even sure it's worth the TODO.